### PR TITLE
Summary badge not shown when the pipeline times out

### PR DESCRIPTION
**Pull request by vsebe**
Parameterize timeout period and always show downstream jobs status.

[ci skip]

Signed-off-by: Violeta Sebe <vsebe@ca.ibm.com>

### CODE REVIEW

**Comment by vsebe** [comment]
[Pipeline-Build-Test-All](https://ci.eclipse.org/openj9/view/Sandbox/job/vs_Pipeline-Build-Test-All/24/)

**Comment by vsebe** [comment]
@jdekonin please review, thank you!

File: buildenv/jenkins/jobs/pipelines/Pipeline-Build-Test-All [hunk 1]
```diff
@@ -355,37 +361,40 @@ def get_summary_table() {
             // table cell
             def innerTable = "<table><tbody>"
 
-            def pipelineName = get_pipeline_name(spec, release)
-            def build = pipelineBuilds.get(pipelineName)
-            def downstreamJobNames = buildFile.get_downstream_job_names(spec, release)
+            // check if this release is supported for this spec
+            if (BUILD_SPECS.get(spec).contains(release)) {
```
Comment (pshipton) [inline buildenv/jenkins/jobs/pipelines/Pipeline-Build-Test-All]: This change doesn't seem to fit the description of the PR. Should it be made, and described, in a separate PR?

Comment (vsebe) [reply buildenv/jenkins/jobs/pipelines/Pipeline-Build-Test-All]: moved to another PR: see https://github.com/eclipse/openj9/pull/5376

**Pull request merged**